### DISCUSSION TITLE
Ajuda na instalação no servidor hostinger

### DISCUSSION THREAD

**Discussion by DataPowerTI** [RamonSilva20/mapos#2110] [Q&A]
Bom dia pessoal, preciso da ajuda de vcs.
Fiz a instalação do mapos conforme fala aqui e em alguns videos que eu vi na internet, td normal.
Subi pra uma pasta na hostinger, acessei, coloquei os dados do banco de dados, usuario e senha, até aí td normal, porém, quando insiro os dados do site pra ir pra tela de login acontece isso:
Eu coloco: https://os.datapowerti.com.br
E duplica; https://os.datapowerti.com.br/os.datapowerti.com.br/index.php/login
Se eu retiro a parte duplicada, os.datapowerti ele aparece os campos de login e senha, porém quando eu insiro ele redireciona pro site duplicado.
Alguém ja viu isso?

**Comment by almron** [comment]
Veja primeiro como foi configurado o arquivo /application/config/config.php.

Na linha 42
`$config['base_url'] = 'https://os.datapowerti.com.br';`

Na linha 54, permaneça do jeito que está:
`$config['index_page'] = 'index.php';`

Na Hostinger, veja se foi configurado o redirecionamento do site. Por lá, você não deveria ter qualquer redirecionamento configurado.

**Reply by DataPowerTI** [reply]
![image](https://user-images.githubusercontent.com/132153532/235320376-dbea7cf6-16f0-4759-be0d-0e14e2f4eeb3.png)

Esta assim, da maneira que vc disse.
Onde posso ver sobre esse redirecionamento na hostinger? sabe me dizer?

**Comment by microreis** [comment]
@DataPowerTI  a maneira que esta o seu código vai sempre duplicar:
seu código `$config['base_url'] = 'os.datapowerti.com.br';`
o correto a ser colocado é:
`$config['base_url'] = 'https://os.datapowerti.com.br';` o http ou https tem que ser colocado.
assim também como o @almron colocou.

**Reply by DataPowerTI** [reply]
Mt obrigado, não tinha me atentado a esse detalhe que o @almron disse.
Deu certo, agora foi pra tela de login, porém o usuario e senha que eu cadastrei não entra, da incorreto....
Apaguei o diretório e fiz a instalação novamente, porém da usuario e senha incorretos....
Existe uma recuperação de senha?

Editado: Consegui alterar a senha atraves do banco de dados, mas mesmo assim não sei pq deu esse erro, maspor enquanto, mto obrigado pela ajuda pessoal!
Ou fiz alguma coisa de errado novamente?
Coloquei a senha no padraõ maiúscula, minúscula, numero e caractere especial.
